src/main/kotlin/com/ecwid/apiclient/v3/impl/ProductVariationsApiClientImpl.kt

```diff
@@ -17,6 +19,9 @@ internal class ProductVariationsApiClientImpl(
 	override fun deleteVariationImage(request: ProductVariationImageDeleteRequest) =
 		apiClientHelper.makeObjectResultRequest<ProductVariationImageDeleteResult>(request)
 
+	override fun uploadProductVariationImageAsync(request: ProductVariationImageAsyncUploadRequest) =
```

Comment:
nit: let we move this method before deleteVariationImage? to have uploadVariationImage and uploadProductVariationImageAsync together